src/main/kotlin/me/hubertus248/deployer/instance/controller/InstanceController.kt

```diff
@@ -0,0 +1,75 @@
+package me.hubertus248.deployer.instance.controller
+
+import me.hubertus248.deployer.data.entity.InstanceKey
+import me.hubertus248.deployer.security.IsAdmin
+import me.hubertus248.deployer.service.InstanceManagerService
+import me.hubertus248.deployer.service.InstanceService
+import org.springframework.stereotype.Controller
+import org.springframework.ui.Model
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.servlet.view.RedirectView
+
+//todo rewrite
+@Controller
+class InstanceController(
+    private val instanceManagerService: InstanceManagerService,
+    private val instanceService: InstanceService
+) {
+
+    @IsAdmin
+    @GetMapping("/newApp")
```

Comment:
same as above

Reply:
as stated above, this class is `//todo rewrite`

Reply:
"As per my last email" vibes